src/dlgAboutDialog.cpp

```diff
@@ -84,8 +84,8 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
         }
 
         // Repeat for other text, but we know it will fit at given size
-        QString sourceCopyrightText = QChar(169) % QString(" Mudlet makers 2008-") % QString(__DATE__).mid(7);
-        QFont font("DejaVu Serif", 16, QFont::Bold | QFont::Serif | QFont::PreferMatch | QFont::PreferAntialias);
+        QString sourceCopyrightText = QChar(169) % QStringLiteral(" Mudlet makers 2008-") % QStringLiteral(__DATE__).mid(7);
```

Comment:
Actually, because the source code is in UTF-8 we could put the `©️` symbol directly into the main text (though it will look weird in some older git instances....)

Also that hackery I've done to get the end year span to be the compilation year (so we do not have to keep changing it every Jan 1) really upsets some package creators who want [*Reproducible Builds*](https://wiki.debian.org/ReproducibleBuilds)!

Reply:
Yep:

![workspace 1_124](https://user-images.githubusercontent.com/110988/39855520-82fd1dd4-542c-11e8-8acf-6bdc5423f682.png)

Happy to take out that complicated-looking bit of code tbh.